test/jest/default-request-executor.test.js

```diff
@@ -0,0 +1,275 @@
+const DefaultRequestExecutor = require('../../src/default-request-executor');
+
+function buildMockResponse(response) {
+  response = response || {};
+  response.headers = response.headers || {};
+  Object.assign(response.headers, {
+    get: (header) => {
```

Comment:
What is this for?

Reply:
This is mocking out [Headers.get](https://developer.mozilla.org/en-US/docs/Web/API/Headers/get)